dev-util/coverity/Manifest

```diff
@@ -0,0 +1,2 @@
+DIST cov-analysis-linux-2017.07.tar.gz 480416104 BLAKE2B 7c3eb4fc770d51811cb3ffe46d5d20051c6ac9b80d73630cae0615e7050c57d8a84cb8d53f5ef07bfd13c9acd26cb929e1556648fdd16eea1ddaeb7548cd1045 SHA512 c93034b26af784fd6ea205710631df36a67d1bab11a3994e024cead301eac564108175c1a0825c9fb920aaef86225d4692d72e620245bc5f456c38aef9b2e84e
```

Comment:
Please shorten the summary line to 69 chars (it's right if github doesn't cut it ).

Reply:
Fixed